schema_utils_test.py

```diff
@@ -20,6 +20,7 @@
 
 import inspect
 
+from core.domain.email_manager import NOTIFICATION_EMAIL_LIST_SCHEMA
```

Comment:
Sorry, I forgot one thing. This should be "from core.domain import email_manager", then refer to email_manager.NOTIFICATION_EMAIL_LIST_SCHEMA.

Reply:
got it